### ISSUE TITLE
user passwords

### ISSUE THREAD

**Issue by nzyeehaa** [datenstrom/yellow#477]
https://github.com/datenstrom/yellow/issues/6#issuecomment-48110052 no longer exists.
https://github.com/datenstrom/yellow/issues/6#issuecomment-48110052 no longer exists
https://github.com/datenstrom/yellow/issues/6#issuecomment-62719549 no longer exists

things have moved on since 2014, 
but the new password reset tool doesnt send email...
add new user tool doesnt send email...

Are there specific settings that are required on a server? I Imagine it uses php mail, and that is working fine on my server. but yellow still doesnt send emails

**EDIT**
with "EditLoginRestriction 1" above details apply.
with "EditLoginRestriction 0" i have successfully setup new user and edited password and received all relevant emails to both admin and new user

So this may be setup as a security "feature" or an oversight and leaves users unable to edit their passwords.
Should the "forgot your password?" link be disabled/removed if "EditLoginRestriction 1"
OR
should "forgot your password" still work if we are not currently accepting new users?

**Comment by markseu** [comment]
Welcome back, @nzyeehaa. What happens if you use a new/unmodified Datenstrom Yellow, can you then reset passwords or create a new user account? I would check if all your email addresses are correct. Additionally, I would check if there are any errors in file `system/extensions/yellow.log`.

In my experience, the setting `EditLoginRestriction: 1` has no impact on resetting passwords.

**Comment by nzyeehaa** [comment]
Hi @markseu , thanks love all the changes to yellow

Good points...
yes - email address is correct.
no - no errors in the yellow.log

I installed a new version, setup a second user and tested - works as advertised.
disabled the _new user_  function and tested - works as advertised.

`there must be a problem in my setup somewhere. `

**Before posting**, 
I set up a third user on the website in question then disabled _new user_ function and no emails were sent when password reset was used.

**Now...**
I noted that the user trying to change password still has a "pending" in user.ini
set _pending: none_
added a new user, disabled add new user function as before,
password reset process worked perfectly!

only two things that are different 
- ran updates for latest changes to image and contact 
- deleted pending in "user.ini"
